nixos/modules/programs/alvr.nix

```diff
@@ -0,0 +1,33 @@
+{ config, pkgs, lib, ... }:
```

Comment:
I do not think this module is usable unless you include it in `nixos/modules/module-list.nix`